templates/cookie-notice.php

```diff
@@ -2,35 +2,28 @@
 
 namespace Netzstrategen\CoreStandards;
 
+if (has_action('core_standards/cookie_notice/css_extend')) {
+  echo '<style>';
+  do_action('core_standards/cookie_notice/css_extend');
```

Comment:
Does this still make sense? — Existing styling needs to be migrated anyway to this new structure…?

Reply:
As the action already existed before, I thought it might be helpful to leave it there.